### PR TITLE
Save settings implementation

### PR DESCRIPTION
**Pull request by Martium**

### CODE REVIEW

File: Martium.TravelInfo/Forms/TravelInfoForm.cs [hunk 1]
```diff
@@ -9,22 +13,26 @@ namespace Martium.TravelInfo.Forms
     public partial class TravelInfoForm : Form
     {
         private readonly TravelInfoRepository _travelInfoRepository;
+        private TravelInfoSettingsModel _travelInfoSettingsModel;
+
 
         public TravelInfoForm()
         {
             _travelInfoRepository = new TravelInfoRepository();
+            _travelInfoSettingsModel = new TravelInfoSettingsModel();
```
Comment (erinev) [inline Martium.TravelInfo/Forms/TravelInfoForm.cs]: For models initialization is not required, only for classes which has methods so you can remove this line

File: Martium.TravelInfo/Forms/TravelInfoForm.cs [hunk 2]
```diff
@@ -57,17 +61,186 @@ private void InitializeMap()
...
+            DepartureCountryTextBox.Text = _travelInfoSettingsModel.DepartureCountry;
+            DepartureAddressTextBox.Text = _travelInfoSettingsModel.DepartureAddress;
+            PricePerKm.Text = _travelInfoSettingsModel.PricePerKm.ToString(CultureInfo.InvariantCulture);
+            AdditionalDistanceInKmTextBox.Text =
+                _travelInfoSettingsModel.AdditionalDistanceInKm.ToString(CultureInfo.InvariantCulture);
         }
 
+        private void DepartureCountryTextBox_TextChanged(object sender, System.EventArgs e)
+        {
+            if (DepartureCountryTextBox.Text == "LTU")
```
Comment (erinev) [inline Martium.TravelInfo/Forms/TravelInfoForm.cs]: converting LTU -> Lietuva should be done in `TravelInfoForm_Load` event single time `_TextChanged` event should be used only for validation/buttons enabling (which is not needed for this text box)

File: Martium.TravelInfo/Forms/TravelInfoForm.cs [hunk 3]
```diff
@@ -57,17 +61,186 @@ private void InitializeMap()
...
+        }
+
+        private static void ShowErrorDialog(string message)
+        {
+            MessageBox.Show(message, "Klaidos pranešimas", MessageBoxButtons.OK, MessageBoxIcon.Error);
+        }
+
+
+
+        private void SaveDepartureAddressButton_Click(object sender, EventArgs e)
```
Comment (erinev) [inline Martium.TravelInfo/Forms/TravelInfoForm.cs]: event methods should be above private non event methods (move up to other event methods)

Comment (erinev) [reply Martium.TravelInfo/Forms/TravelInfoForm.cs]: same applies for methods below:
AdditionalDistanceInKmTextBox_Validating
PricePerKm_Validating
SavePricePerKmButton_Click
SaveAdditionalDistanceInKmButton_Click

**Review by erinev — CHANGES_REQUESTED** [review]

File: Martium.TravelInfo/Forms/TravelInfoForm.cs (hunk 3 above)
Comment (Martium) [reply Martium.TravelInfo/Forms/TravelInfoForm.cs]: yeah i know this and was thinking to as you how make helpers methods? as was in funeral service

Comment (erinev) [reply Martium.TravelInfo/Forms/TravelInfoForm.cs]: google c# region 

but you don't need to create region just to change methods placements, region is optional thing 
my request was reorder methods so event methods would be first below constructor and only after them other private helper methods (if you want you can move helpers to region)

**Review by erinev — APPROVED** [review]

**Pull request merged**